azurerm/resource_arm_app_service.go

```diff
@@ -43,7 +43,10 @@ func resourceArmAppService() *schema.Resource {
 							Required:         true,
 							DiffSuppressFunc: ignoreCaseDiffSuppressFunc,
```

Comment:
FYI I'm not sure if this should be here (`DiffSuppressFunc: ignoreCaseDiffSuppressFunc`), but I didn't change it for this PR.